src/components/Home/countryItem.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] I request that you re-enable the linters and fix them instead. They help to create good quality code that is also well formatted.